src/test/resources/log4j2-test.properties

```diff
@@ -0,0 +1,41 @@
+# Allows this configuration to be modified at runtime. The file will be checked every 30 seconds.
```

Comment:
With this file, we can remove the logback configurations in this folder now.

Reply:
Removed.